beacon-chain/p2p/types/types.go

```diff
@@ -46,6 +58,22 @@ func (s *SSZUint64) UnmarshalSSZ(buf []byte) error {
 	return nil
 }
 
+// SSZUint64 is a uint64 type that satisfies the fast-ssz interface.
+type SSZBytes []byte
+
+// HashTreeRoot hashes the uint64 object following the SSZ standard.
+func (s *SSZBytes) HashTreeRoot() ([32]byte, error) {
+	hasher := ssz.NewHasher()
+	hasher.PutBytes(*s)
+	return hasher.HashRoot()
+}
+
+// HashTreeRootWith hashes the uint64 object with the given hasher.
+func (s *SSZBytes) HashTreeRootWith(hh *ssz.Hasher) error {
+	hh.PutBytes(*s)
```

Comment:
Not sure what I should be doing here.

Reply:
Can we add some tests for this to increase confidence? 

We can look at other ssz tools to come up with input / output data.

Reply:
Check out these from the spec test repo: https://github.com/ethereum/eth2.0-spec-tests/tree/master/tests/general/phase0/ssz_generic/uints

Reply:
Added!